src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1274,14 +1274,28 @@ map.defaults = {
         shown = false,
     },
     prompt_test_patterns = {"^%[?%a*%]?&lt;.*&gt;", "^%[.*%]%s*&gt;", "^%w*[%.?!:]*&gt;", "^%[.*%]", "^[Hh][Pp]:.*&gt;"},
-    custom_exits = {},  -- format: short_exit = {long_exit, reverse_exit, x_dif, y_dif, z_dif}
+    custom_exits = {
+        eu = {"eastup", "westdown", 1, 0, 0},
```

Comment:
Shouldn't have these combined exits also have a difference in the `z` axis? The example has this.